test/multidevice.cpp

```diff
@@ -93,12 +93,13 @@ void CommunicationTest::resetDstBuffers() {
 // It compares the given (possibly sharded) output with the result of the Fusion
 // run on a single device with the given (possibly sharded) inputs
 void PipelineTest::validate() {
-  // execute the fusion on one device without pipeline scheduling
-  auto fusion_copy = std::make_unique<Fusion>(*runtime->completeFusion());
-  unshard(fusion_copy.get());
-  FusionExecutorCache unsharded_fec(std::move(fusion_copy));
-  auto ref_unsharded_outputs =
-      unsharded_fec.runFusionWithInputs(unsharded_inputs);
+  if (ref_unsharded_outputs.empty()) {
```

Comment:
https://github.com/NVIDIA/Fuser/pull/1853#discussion_r1523694537

Your comment makes a lot of sense. 
Checking if it's empty is the way to signal that we want to validate with prescribed tensors. Do you suggest having an explicit flag?